src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialJsPage.java

```diff
@@ -52,4 +58,31 @@ public ContentReviewModule getReviewModule() {
         }
         return contentReviewModule;
     }
+
+    public boolean isTestElementVisible() {
+        boolean isVisible = false;
+        try {
+            wait.forElementVisible(testElement, 3, 1);
+            isVisible = true;
+        } catch (TimeoutException $e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00117)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)